h/static/scripts/site.js

```diff
@@ -24,6 +25,10 @@ page('/groups/new', function () {
   new CreateGroupFormController(document.body);
 });
 
+page('/login', function() {
+  new FormSelectOnFocusController(document.body);
```

Comment:
This issue and the other one just means there should be a `'use strict';` at the top of the file.